.env.example

```diff
@@ -5,10 +5,10 @@
 # Optional: (Default value: 587) Port address of the SMTP server to use.
 #SMTP_PORT=
 
-# Mandatory: Username to authenticate with.
+# Optional: Username to authenticate with.
 #SMTP_USERNAME=
 
-# Mandatory: Password of the SMTP user. (Not needed if SMTP_PASSWORD_FILE is used)
+# Optional (Mandatory if SMTP_USERNAME is set): Password of the SMTP user. (Not needed if SMTP_PASSWORD_FILE is used)
```

Comment:
This is correct, but in the code changes you are handling SMTP_PASSWORD as optional.